### ISSUE TITLE
AGW: OVS update to 2.8.10-2

### ISSUE THREAD

**Issue by pshelar** [magma/magma#6157]
This build has OVS fix required for inbound roaming. Ref to OVS patches for details.

We meed following changes:
1. update vagrant binaries to new version.
2. update packages in artifactory to this newer OVS package
3. verify AGW installation installs this newer OVS package.

**Comment by uri200** [comment]
The newer versions should look like this 

```


magma@agw:~/uri$ dpkg --list |grep openv
ii  libopenvswitch:amd64                      2.8.10-2                                    amd64        Open vSwitch common components
ii  openvswitch-common                        2.8.10-2                                    amd64        Open vSwitch common components
ii  openvswitch-datapath-dkms                 2.8.9-1                                     all          Open vSwitch datapath module source - DKMS version
ii  openvswitch-datapath-module-4.9.0-9-amd64 2.8.10-2                                    amd64        Open vSwitch Linux datapath kernel module
ii  openvswitch-datapath-source               2.8.10-1                                    all          Open vSwitch datapath module source - module-assistant version
ii  openvswitch-switch                        2.8.10-2                                    amd64        Open vSwitch switch implementations
ii  python-openvswitch                        2.8.10-2                                    all          Python bindings for Open vSwitch

```

**Comment by pshelar** [comment]
to manually build these packages:
```
git clone https://github.com/magma/magma
cd magma/lte/gateway/release/
bash -x build-ovs.sh
ls open*-2_*.deb. # install these packages using `dpkg`
```

**Comment by pshelar** [comment]
its merged to 1.5
